java-frontend/src/main/java/org/sonar/java/bytecode/cfg/Instruction.java

```diff
@@ -115,7 +115,7 @@ public boolean hasReturnValue() {
     return Type.getMethodType(fieldOrMethod.desc).getReturnType() != Type.VOID_TYPE;
   }
 
-  private boolean isInvoke() {
+  public boolean isInvoke() {
```

Comment:
missing annotation @VisibleForTesting (is public required though ? isn't package protected sufficient ?)

Reply:
`public` was required, but indeed I think it will be better to remove `org.sonar.java.bytecode.cfg` package and move everything into `org.sonar.java.bytecode.se`

Reply:
or rather not :)